app/Models/Forum/Topic.php

```diff
@@ -419,6 +419,11 @@ public function setTopicLastPosterColourAttribute($value)
         $this->attributes['topic_last_poster_colour'] = ltrim($value, '#');
     }
 
+    public function setTopicTitleAttribute($value)
+    {
+        $this->attributes['topic_title'] = preg_replace('/(^\pZ+|\pZ+$)/u', '', trim($value));
```

Comment:
When `/u` is enabled `\s` becomes `\t\n\f\r\p{Z}` then `trim` shouldn't be needed anymore